CHANGELOG.md

```diff
@@ -7,9 +7,13 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+### Added
+- `Field::try_from_rng` method.
```

Comment:
This is a change to the crate, not an addition, because it adds a new method to the `Field` trait without a default implementation.